libs/payments/eligibility/project.json

```diff
@@ -8,6 +8,7 @@
       "executor": "@nx/js:tsc",
       "outputs": ["{options.outputPath}"],
       "options": {
+        "rootDir": ".",
```

Comment:
Since landing #15925 we shouldn't need to add `rootDir` anymore. (From what I remember the original reason `rootDir` was added isn't an issue anymore. That PR should have more info)

Reply:
Unfortunately still needed.